recipes/kmc-smudge-pairs/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "kmc_smudge_pairs" %}
+{% set version = "3.1.2rc2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gkanogiannis/KMC/archive/v{{ version }}.tar.gz
+  sha256: ae33294e085c6d68a31dab945dbc41fa35b1019fe3616604637a46b08fdfc38d
+
+build:
+  skip: true  # [not linux]
+  script: bash build_shared.sh
+  number: 0
+  run_exports:
+    - zlib
+    - bzip2
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+    - make
+    - libzlib
+    - anaconda::zlib-devel-amzn2-aarch64
+    - bzip2
+  host:
+    - zlib
+    - bzip2
```

Don't have them at all here. Once, this is removed, I'm happy to review the error message and give guidance on how to continue.
```suggestion
  host:
    - zlib
    - bzip2
```